tests/integration_test/services_test.py

```
@@ -9,7 +9,7 @@
 from datetime import datetime, timedelta, date
 from io import BytesIO
 from unittest import mock
-from unittest.mock import patch, AsyncMock
+from unittest.mock import patch, AsyncMock, MagicMock
```

Ruff (0.12.2)

`unittest.mock.AsyncMock` imported but unused

Remove unused import

(F401)